package.json

```diff
@@ -1,23 +1,39 @@
 {
-  "name": "purdue-laundry-api",
+  "name": "purduelaundry-server",
   "version": "1.0.0",
-  "description": "Server for parsing things in my first attempt at NodeJS",
+  "description": "The API Server for ACM SigApp's Purdue Laundry app",
   "main": "index.js",
+  "repository": "https://github.com/purdue-acm-sigapp",
   "scripts": {
-    "start": "node app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "docker-compose build",
+    "start": "docker-compose up",
+    "stop": "docker-compose down",
+    "start:vanilla": "pm2 start processes.json --no-daemon",
+    "test": "npm run mocha && npm run eslint",
+    "mocha": "mocha test/cases/*",
+    "eslint": "node_modules/.bin/eslint server/*.js app.js"
   },
-  "author": "",
+  "author": "Purdue ACM SigApp <acm@purdue.edu>",
```

Comment:
cs.purdue.edu? 

I forget what works anymore.

Reply:
I literally have no idea lmao I'll ask jordan and update it